cmd/digger/main.go

```diff
@@ -170,6 +177,70 @@ func gitLabCI(lock utils.Lock) {
 	println("Commands executed successfully")
 }
 
+func azureCI(lock locking.Lock) {
+	println("Using Azure.")
```

Comment:
better log message like 

```suggestion
	fmt.Println("> Azure CI detected")
```